README.md

```diff
@@ -1,8 +1,17 @@
-# Schug 
+# Schug
 
-Schug :stew: is a service that gather data about genes, transcripts and exons from multiple sources and merge the 
+Schug :stew: is a service that gather data about genes, transcripts and exons from multiple sources and merge the
 information. There is a [REST API][rest-api] with relevant endpoints.
 
+## Test the app using Docker
+
+You can test Schug on your local computer using Docker. Make sure you have [Docker][docker] installed and type the following command in a terminal window: `docker-compose up`
```

Comment:
Don’t you have to be in the root for or point to the docker compose file?

Reply:
Ah yes, I'll add it!

Reply:
I guess you also pull the image from our Docker Hub, but we don't have a prod image yet, so I guess it's better to write to download it and go to the root